calin/nz/_posts/2022-07-24-the-move.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: The Move
+author: Călin Cașcaval
+---
+
+Moving house across continents is not an easy feat: although we took the opportunity
+to somewhat slim down when we moved houses in Oct. 2020, it was too easy to just
+"take it" when we mved one mile across town! This time however, we agreed that it
```

Comment:
moved*

Reply:
it's also not 100% clear to me the first time through this sentence what "it" means in the phrase "take it" (though it becomes clear from context later that it is the stuff)

Reply:
thanks!